src/gaggiuino.ino

```diff
@@ -683,13 +683,12 @@ bool isBoilerFillPhase(unsigned long elapsedTime) {
 
 bool isBoilerFull(unsigned long elapsedTime) {
   bool boilerFull = false;
-  if (elapsedTime > BOILER_FILL_START_TIME + 1000) {
+  if (elapsedTime > BOILER_FILL_START_TIME + 1000UL) {
     boilerFull =  (previousSmoothedPressure - currentState.smoothedPressure > -0.05f)
                 &&
                   (previousSmoothedPressure - currentState.smoothedPressure < 0.05f);
```

Comment:
Do you actively want to keep this disconnected from the definitions of `isPressureRising` and `isPressureFalling` or could this be rephrased as `boilerFull = !(currentState.isPressureRising || currentState.isPressureFalling);`?

Reply:
Yes, these can change independently of the other definitions, mixing them up can surely create confusion later as tuning those for one feature(ex predictive scales) will fuck up the other one.